ftests/nuxeo-platform-explorer-ftests/src/test/java/org/nuxeo/functionaltests/explorer/AbstractExplorerTest.java

```diff
@@ -53,24 +57,40 @@
  */
 public abstract class AbstractExplorerTest extends AbstractTest {
 
+    protected static String READER_USERNAME = TEST_USERNAME;
+
+    protected static String MANAGER_USERNAME = "apidocmanager";
+
+    public static LoginPage getLoginPageStatic() {
+        return get(NUXEO_URL + "/logout", LoginPage.class);
+    }
+
     protected void doLogin() {
-        getLoginPage().login(TEST_USERNAME, TEST_PASSWORD);
+        getLoginPageStatic().login(TEST_USERNAME, TEST_PASSWORD);
+    }
+
+    protected static void loginAsAdmin() {
+        getLoginPageStatic().login(ADMINISTRATOR, ADMINISTRATOR);
     }
 
-    protected void doLogout() {
+    protected static void doLogout() {
         // logout avoiding JS error check
         driver.get(NUXEO_URL + "/logout");
     }
```

Comment:
Should we make `doLogin` static too? I think it could be interesting to have the same kind of method (static or not) for `doLogin` and `doLogout`, wdyt?

Reply:
Well i initially made it static, and then made it unstatic to use in the ApidocManager test extending the admin one...